documents/openapi/openapi.yaml

```diff
@@ -1818,6 +1848,14 @@ components:
     Id:
       type: string
       format: uuid
+    MobileApp:
+      type: string
+      nullable: true
+      enum:
+        - VA_FLAGSHIP_APP
+        - VETEXT
```

Comment:
not sure this should be an enum? the default and then the message about optional seems fair

Reply:
I'm not sure about this.  It already exists for the /v2/notifications/push route.

Reply:
Leave it as it then!